libbeat/processors/registered_domain/docs/registered_domain.asciidoc

```diff
@@ -9,7 +9,8 @@ The `registered_domain` processor reads a field containing a hostname and then
 writes the "registered domain" contained in the hostname to the target field.
 For example, given `www.google.co.uk` the processor would output `google.co.uk`.
 In other words the "registered domain" is the effective top-level domain
-(`co.uk`) plus one level (`google`).
+(`co.uk`) plus one level (`google`). Optionally, it can store the rest of the
+domain, the `subdomain` into another target field.
 
 This processor uses the Mozilla Public Suffix list to determine the value.
```

Comment:
Can you please update the example below with the new config option to encourage use.

Reply:
Sounds good.

For the `top_level_domain` I'll just go ahead and create an issue for it. If I have time to add it to the processor after the rest of the ECS upgrade I'll go ahead and tackle it.